docs/usage/development/integration.md

```diff
@@ -30,6 +30,20 @@ npm install tauri --save
   You can install Tauri as both a local and a global dependency, but we recommend installing it locally.
 </Alert>
 
+If you decide to use Tauri as a local package with npm, you will _have to_ define custom scripts to your package.json:
+
+```js title=package.json
+{
+  // This content is just a sample
+  "scripts": {
+    "tauri:init": "tauri init",
```

Comment:
I think we just leave a `"tauri": "tauri"` script then the user can run `npm run tauri init` `npm run tauri build`.